universal-login-relayer/lib/http/routes/authorisation.ts

```diff
@@ -67,7 +64,7 @@ export default (authorisationService : AuthorisationService, provider: any) => {
       walletContractAddress: asString,
       body: asObject({
         publicKey: asString,
-        signature: asSignature
+        signature: asString
```

Comment:
I would keep `asSignature` - which would verify that field is a hex string of proper length.